src/jemalloc.c

```diff
@@ -1322,6 +1322,17 @@ malloc_conf_init_helper(sc_data_t *sc_data, unsigned bin_shard_sizes[SC_NBINS],
 				} while (vlen_left > 0);
 				CONF_CONTINUE;
 			}
+			if (CONF_MATCH("ncached_max")) {
```

Comment:
How about `tcache_ncached_max` (which is similar to tcache_max intentionally)?

Reply:
Sounds good, modified.